website/docs/r/sql_server.html.markdown

```diff
@@ -47,9 +47,9 @@ The following arguments are supported:
 
 * `version` - (Required) The version for the new server. Valid values are: 2.0 (for v11 server) and 12.0 (for v12 server).
 
-* `administrator_login` - (Required) The administrator login name for the new server. Changing this forces a new resource to be created.
+* `administrator_login` - (Required) The administrator login name for the new server. Changing this forces a new resource to be created. This value is immutable, so if you change it, terraform will have to destroy and recreate the server.
```

Comment:
We already state this with the phrase `Changing this forces a new resource to be created.` so i don't think we need to repeat it again?
```suggestion
* `administrator_login` - (Required) The administrator login name for the new server. Changing this forces a new resource to be created.
```

Reply:
Yes, apparently I missed words right in front of me. Reverted, but made the original sentence bold.